pkg/base/config.go

```diff
@@ -361,6 +361,9 @@ type RaftConfig struct {
 	// without acknowledgement. With an average entry size of 1 KB that
 	// translates to ~4096 commands that might be executed in the handling of a
 	// single raft.Ready operation.
+	//
+	// This setting is used both by sending and receiving end of Raft messages, in
+	// order to minimize dropped messages on the receiving end.
```

Comment:
Obv, this works only if both ends are configured the same way (either use default values, or the env variable for this setting matches). Should we say it here, or it's always assumed?

Reply:
It's worth mentioning.